package.json

```diff
@@ -30,8 +30,8 @@
     "storybook:dist": "build-storybook -c .storybook -o dist/storybook",
     "storybook": "start-storybook -p 9001",
     "sassdoc": "sassdoc {packages/bpk-mixins/src/**/*,packages/bpk-svgs/dist/*,packages/bpk-tokens/tokens/base.default}.scss -d dist/sassdoc -v --strict",
-    "docs": "npm run submodules:pull && BPK_BUILT_AT=$( date -u +%s ) webpack-dev-server --open",
-    "docs:dist": "npm run submodules:pull && npm run clean:dist && NODE_ENV=production BPK_BUILT_AT=$( date -u +%s ) webpack --progress --display=minimal --bail && npm run sassdoc",
+    "docs": "npm run submodules:init && BPK_BUILT_AT=$( date -u +%s ) webpack-dev-server --open",
```

Comment:
If I understand correctly this will make it impossible to test with the latest version of the submodules, unless we first update and merge the submodules, and rebase the branch. Is that right?

Reply:
So you could still test with the latest ones by running `npm run submodules:pull`

Reply:
Happy to close this though if we don't think it improves the current process

Reply:
But if the docs command runs init again it will revert back to the current version no? I'm ok with the change if we can somehow still test with the latest locally

Reply:
I see what you mean... 🤔